_data/library.yml

```diff
@@ -1095,3 +1095,57 @@
     - name: run-pex-as-resource
       blurb: run a Python (pex) script as a data source
       description: Run a python script with dependencies packaged together as a local-exec provisioner on a null resource in a portable way.
+
+- name: "Influx GCP"
+  clouds:
+    - GCP
+  icons:
+    - title: Terraform
+      image: icon-terraform.png
+    - title: Bash
+      image: icon-bash.png
+  aws_url: "https://github.com/gruntwork-io/terraform-google-influx"
```

Comment:
Is there a `gcp_url` equivalent?

Reply:
Nope, I think the cloud provider is presented with tags and technologies as icons